migration/src/main/resources/migrations/ensure-users-are-providers.xml

```diff
@@ -10,18 +10,14 @@
     <changeSet id="ensure-orders-creators-are-providers" author="IMB">
```

Comment:
You'll want to change this changeSet id to reflect the file name

Reply:
This was changed in the commit that followed!

Reply:
Are you sure?  The file name has changed, but did you change the id within the <changeSet id="..."> tag inside the file?